WcaOnRails/Envfile

```diff
@@ -19,14 +19,11 @@ variable :WCA_LIVE_SITE, :boolean, default: false
 
 # ROOT_URL is used when generating emails.
 # Trick to discover the port we're set to run on from
-# http://stackoverflow.com/a/13839447. However, Rails::Server isn't always
-# defined (during rake tasks, for instance).
```

Comment:
I played around a bit and couldn't find an environment in which `Rack::Server` is not defined, but it's very likely we're going to run into something weird on production with this change. Everyone please keep an eye out for this!